app/src/components/trayIcon.ts

```diff
@@ -39,14 +34,6 @@ function createTrayIcon(inpOptions, mainWindow) {
 
     appIcon.on('click', onClick);
 
-    mainWindow.on('show', () => {
-      appIcon.setHighlightMode('always');
-    });
-
-    mainWindow.on('hide', () => {
-      appIcon.setHighlightMode('never');
-    });
```

Comment:
> Is this not needed anymore?

@jiahaog it's dead [since Electron 6](https://www.electronjs.org/docs/api/breaking-changes#planned-breaking-api-changes-60):

> Under macOS Catalina our former Tray implementation breaks. Apple's native substitute doesn't support changing the highlighting behavior.
>
> ```js
> // Deprecated
> tray.setHighlightMode(mode)
> // API will be removed in v7.0 without replacement.
> ```